datastore/db/queries/picture/__init__.py

```diff
@@ -308,6 +308,30 @@ def get_picture_set_pictures(cursor, picture_set_id: str):
     except Exception:
         raise GetPictureError(f"Error: Error while getting pictures for picture_set:{picture_set_id}")
 
+def get_pictures_with_picture_seed(cursor, picture_set_id: str):
```

Comment:
I think the name of this function and the description of it might be a bit confusing. Even though I know the verified pictures are the one with a picture_seed, the future devs might not.

Reply:
Yes I agree, what about `get_validated_pictures`  ? and add the description you suggested "This functions select pictures from a picture set that have been validated. Therefore, there should exists picture_seed entity for this picture." ?